src/braket/pennylane_plugin/braket_device.py

```diff
@@ -605,6 +606,61 @@ def execute_and_gradients(self, circuits, **kwargs):
         res = res[0] if len(res) == 1 and active_jac else res
         return res, jacs
 
+    def _is_single_qubit_01_frame(self, f):
+        if self._device.arn == "arn:aws:braket:eu-west-2::device/qpu/oqc/Lucy":
+            return "drive" in f
+        else:
+            raise RuntimeError(
+                f"Single-qubit drive frame for pulse control not defined for "
+                f"device {self._device.arn}"
+            )
+
+    def _is_single_qubit_12_frame(self, f):
+        if self._device.arn == "arn:aws:braket:eu-west-2::device/qpu/oqc/Lucy":
+            return "second_state" in f
+        else:
+            raise RuntimeError(
+                f"Single-qubit drive frame for pulse control not defined for "
+                f"device {self._device.arn}"
+            )
+
+    def _get_frames(self, filter):
+        return {
+            f: info
+            for f, info in self._device.properties.pulse.dict()["frames"].items()
+            if filter(f)
+        }
+
+    @property
+    def settings(self):
+        frames = self._get_frames(filter=self._is_single_qubit_01_frame)
+        frames_12 = self._get_frames(filter=self._is_single_qubit_12_frame)
+
+        drive_frequencies = [frames[f]["frequency"] * 1e-9 for f in frames]  # Hz to GHz
+        device_info = self._device.properties.paradigm.dict()
+
+        connections = []
+
+        for q1, connected_qubits in device_info["connectivity"]["connectivityGraph"].items():
+            for q2 in connected_qubits:
+                connection = (int(q1), int(q2))
+                connections.append(connection)
+
+        # need to decide how to deal with non-chronological wiring order on rigetti if using
+        wires = [i for i in range(device_info["qubitCount"])]
+
+        second_excitation_freqs = [frames_12[f]["frequency"] * 1e-9 for f in frames_12]  # Hz to GHz
+        anharmonicities = [
+            f01 - f12 for f01, f12 in zip(drive_frequencies, second_excitation_freqs)
+        ]
+
+        return {
+            "qubit_freq": drive_frequencies,
```

Comment:
can we use the same terms as in pennylane?
```suggestion
            "qubit_freq": qubit_freq,
```
same below for `anharmonicity`